packaging/debian_3rdparty/build_tensorflow.sh

```diff
@@ -10,75 +10,30 @@ cd tmp
 
 echo "Building Tensorflow $TENSORFLOW_VERSION"
 
+# NumPy is required by Bazel
+${PYTHON_BIN_PATH} -m pip install numpy
```

Comment:
Why the `numpy` Python package is required? 

Btw, we already demand `numpy` dependency in https://essentia.upf.edu/installing.html#installing-dependencies-on-linux, so we can expect a user to install them before running build scripts.